fix(curriculum): build a city skyline instruction updated (#48010)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md

# --description--

> 1. Turn your `.bb1` element into a flexbox parent so you can center the parts of the building.<br>- hint: You should give `.bb1` a `display` of `flex`

Reviewer: The `hints` are what go in the next section to display when a test fail.

It seems you misinterpreted what was intended with "split in 3 different steps"

please check our documentation and look at how the project works in the curriculum.

assert.equal(bb1Style?.height, '70%');
```

> You should give `.bb1` a `display` of `flex`.

Reviewer: ```suggestion
You should give the `.bb1` element a `display` of `flex`.
```

```js
const bb1Style = new __helpers.CSSHelp(document).getStyle('.bb1');
assert.equal(bb1Style?.display, 'flex');
```

> `.bb1` should have a `flex-direction` property.<br>You should give `.bb1` a `flex-direction` of `column`

Reviewer: ```suggestion
You should give the `.bb1` element a `flex-direction` of `column`.
```

```js
const bb1Style = new __helpers.CSSHelp(document).getStyle('.bb1');
assert.equal(bb1Style?.flexDirection, 'column');
```

> Your `.bb1` should have an `align-items` property.<br>You should give `.bb1` an `align-items` property of `center`

Reviewer: ```suggestion
You should give the `.bb1` element a `align-items` of `center`.
```
